src/pages/billing/List.js

```diff
@@ -40,34 +40,44 @@ const styles = {
   }
 }
 
-const ProjectListItem = ({ project: { projectName, role, creationStatus, message }, isActive }) => {
-  const projectReady = creationStatus === 'Ready'
-  const statusIcon = creationStatus === 'Creating' ?
-    spinner({ size: 16, style: { color: colors.accent(), margin: '0 1rem 0 0.5rem' } }) :
-    h(InfoBox, {
-      style: { color: colors.danger(), margin: '0 1rem 0 0.5rem' }, side: 'right'
-    }, [div({ style: { wordWrap: 'break-word', whiteSpace: 'pre-wrap' } }, [message || 'Error during project creation.'])])
-  const selectableProject = h(Clickable, {
+const ProjectListItem = (() => {
```

Comment:
We don't tend to write self-invoking functions in this codebase. Instead, could you move the actual props for the component up here, where they're easier to find? Additionally, it looks like `selectableProject` and `unselectableProject` don't need anything outside of what's given to them as props; perhaps it would be better to declare them as independent components outside of `ProjectListItem` and keep this as a just ternary? That would let React better optimize rendering.